src/misc/deployments-to-sqs.ts

```diff
@@ -30,6 +31,11 @@ export namespace DeploymentToSqs {
         },
         additionalProperties: true
       },
+      lods: {
+        type: 'array',
+        items: { type: 'string' },
+        nullable: true
```

Comment:
Doesn't this nullable apply to each of the strings? Can each of the strings be null?

Reply:
Hello, I do think this means that the lods property can be missing which is what we want since this property is only be present when generating lods through the AB-Converter